pkg/telemetry/statsworker.go

```diff
@@ -154,7 +157,15 @@ func coalesce(stats []*livekit.AnalyticsStat) *livekit.AnalyticsStat {
 			continue
 		}
 
-		score += stat.Score
+		// only consider non-zero scores
+		if stat.Score > 0 {
```

Comment:
what does a zero score mean?

Reply:
I believe it is just a sanity check. The scorer does not produce 0 (in theory it should not produce 0, but there could always be bugs😊)